pkg/apis/kops/v1alpha1/cluster.go

```diff
@@ -343,3 +346,15 @@ type ClusterZoneSpec struct {
 
 	Egress string `json:"egress,omitempty"`
 }
+
+type EgressProxySpec struct {
+	HTTPProxy     HTTPProxySpec `json:"httpProxy,omitempty"`
+	ProxyExcludes string        `json:"excludes,omitempty"`
```

Comment:
We are going to need some validation around this.  See the validation folder in this pkg.